templates/definition/meter/kostal-plenticore-ng.yaml

```diff
@@ -1,22 +1,22 @@
-template: kostal-plenticore-hw0200
+template: kostal-plenticore-ng
 products:
   - brand: Kostal
     description:
-      generic: Plenticore Hybrid (Gen2)
+      generic: Plenticore Hybrid (nextgen)
```

Comment:
What makes this next?

Reply:
Anwender verstehen das nicht. Generationen von Templates? Huh?

Reply:
Ich persönlich fände nach wie vor irgendwas mit `-gridcharge` am Besten. Von mir aus auch `-gridchargefeature`.

Was sprechendes wäre einfach sinnvoll, verstehe aber natürlich die Einwände von @andig.

Daher die Einigung auf `-nextgen`.

Reply:
Mich interessiert weniger der Name- das ist technishces Detail- sondern v.a. die Beschreibung. Mit nexgen kann ein Anwender nichts anfangen. Ebensowenig Grid Charge- warum sollte er das _nicht_ auswählen? 

Gibts denn keine Aussage von Kostal dazu???